obiba_opal/system.py

```diff
@@ -342,6 +342,56 @@ class RESTService:
     Perform raw web services requests.
     """
 
+    def __init__(self, client: core.OpalClient, accept: str = None, headers = None, verbose: bool = False):
```

Comment:
accept and headers are not used.

Reply:
Removed, they are used when creating a request